### DISCUSSION TITLE
https://penguins-eggs.net/blog/add-giscus-docusaurus

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#2] [General]
# https://penguins-eggs.net/blog/add-giscus-docusaurus

Questo articolo è una libera traduzione di how to add giscus comments to docusaurus.

https://penguins-eggs.net/blog/add-giscus-docusaurus

**Comment by pieroproietti** [comment]
Finally it's possible to comment!!!
